### PR TITLE
Release 0.8.0

### PR DESCRIPTION
**Pull request by konstin**
We have a bunch of changes again, I think it's time to make a release.

### CODE REVIEW

**Comment by IvanKuzavkov** [comment]
Hello, maybe we can include in release memory leak fix [https://github.com/PyO3/pyo3/issues/570]?

**Comment by konstin** [comment]
We can just publish 0.8.1 after the fix. As far as I can see the fix and the associated test hasn't been implemented yet.

**Comment by IvanKuzavkov** [comment]
Sounds good.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.8.0]
+## [0.8.0] - 2018-08-30
```
Comment (kngwyu) [inline CHANGELOG.md]: Let's modify this into the true release date

**Comment by kngwyu** [comment]
Thanks.
I want to add #450 to this release.

**Pull request merged**